sqjobs/tests/tests.py

```diff
@@ -0,0 +1,172 @@
+from django.test import TestCase
+from ..brokers.broker import Standard
+from ..brokers.eager import Eager
+from sqjobs.contrib.django.djsqjobs.beat import Beat
+from sqjobs.contrib.django.djsqjobs.models import PeriodicJob
+from sqjobs.contrib.django.djsqjobs.models import JobStatus
+
+from .fixtures import Adder, Divider
+from .django_fixtures import ResultDivider
+from ..connectors.dummy import Dummy
+
+from datetime import datetime
+import pytz
+
+from mock.mock import patch
+
+
+class DjangoTestCase(TestCase):
+    fixtures = ['sqjobs/tests/djsqjobs_tests-fixture.json']
+
+    def setUp(self):
+        from django.core.management.commands.loaddata import Command
+
+        loaddata_command = Command()
+        loaddata_command.using = 'default'
+        loaddata_command.ignore = None
+        loaddata_command.app_label = None
+        loaddata_command.verbosity = 0
+        loaddata_command.hide_empty = None
+        loaddata_command.loaddata(self.fixtures)
+
+
+class PeriodicJobsTests(DjangoTestCase):
+    beat = Beat(None, 10, True)
+    beat.register_job(Adder)
```

Comment:
Rarely class instance variables that hold status are used for testing, as they produce side effects.

In some of your tests, you actually recreate beat. You might want to use:

```
def setup(self): 
    self.beat = Beat(None, 10, True)
    self.beat.register_job(Adder)
```